constants/Units.java

```diff
@@ -0,0 +1,24 @@
+package constants;
+
+public enum Units {
```

Comment:
What do you think about renaming to `MeasureUnit`?

Reply:
Good idea. changed